src/content/en/updates/2017/06/budget-api.md

```diff
@@ -1,6 +1,9 @@
 project_path: /web/_project.yaml
 book_path: /web/updates/_book.yaml
 description: The Budget API allows developers to perform background actions without notifying users, enabling use cases like silent push.
+blink_components:
```

Comment:
This may break DevSite, we should use the same format we use for other wf meta tags. Specifically `{# wf_blink_components: asdf,qwer,zxcv #}`